src/test/scala/cromwell/engine/WorkflowManagerActorSpec.scala

```diff
@@ -39,42 +42,43 @@ class WorkflowManagerActorSpec extends CromwellTestkitSpec("WorkflowManagerActor
 
       val actual = outputs.map { case (k, WdlString(string)) => k -> string }
       actual shouldEqual Map(HelloWorld.OutputKey -> HelloWorld.OutputValue)
+      Await.result(dataAccess.shutdown(), Duration.Inf)
```

Comment:
could this use a loan pattern for the `DataAccess`?  (Warning: did not test this)

```
with { dataAccess =>
  doStuff()
}

def with[T](work: DataAccess => T): T = {
  val dataAccess = DataAccess()
  work(dataAccess)
  Await.result(dataAccess.shutdown(), Duration.Inf)
}
```